usr.sbin/freebsd-update/freebsd-update.sh

```diff
@@ -2220,7 +2220,7 @@ fetch_run () {
 	# further processing (except forced fetch).
 	if [ $FORCEFETCH -eq 0 ]; then
 		fetch_compare_new_tag_to_current_tag &&
-		    echo "No update required." && return 0
+		    echo "No updates are available to fetch." && return 0
```

Comment:
this is for when someone runs `freebsd-update fetch install` the output will be
`No updates are available to fetch`
`No updates are available to install`

Reply:
If it's looking for "No updates are available to" or "No updates are available" or subset ("No updates") then it'll match early.

Did you think I should alter to something else?  I commented to explain the change or was that a mistake, i.e. not the intent of the "review" comments?

Reply:
> Did you think I should alter to something else?

Let's wait for the maintainers' opinion or the documentation team, they usually have a good suggestion about how the messages should be presented.

Reply:
Perhaps what I meant was that, for example:

https://github.com/freebsd/freebsd-src/blob/b54056e187af4687202ef31dc470ac35b337e91e/usr.sbin/freebsd-update/freebsd-update.sh#L3421

This cron job searches for "No updates needed".